openfisca_france/scripts/parameters/baremes_ipp/merge_ipp_xml_files_with_openfisca_parameters.py

```diff
@@ -0,0 +1,157 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+"""
+Merge IPP XML files with OpenFisca XML parameters.
+
+Keep elements and attributes order from OpenFisca side, replace <CODE> and <BAREME> leaves by IPP ones.
+
+Let the user commit or not the diff using git. The script `../format_parameters.py` can be used to ease the merge.
+"""
+
+
+import argparse
+import os
+import sys
+
+from lxml import etree
+from openfisca_core import legislationsxml
+
+from openfisca_france.france_taxbenefitsystem import FranceTaxBenefitSystem, COUNTRY_DIR
+
+
+parameters_dir_path = os.path.join(COUNTRY_DIR, 'parameters')
+
+
+def replace_children(element, children):
+    for child in element:
+        child.getparent().remove(child)
+    element.extend(children)
+
+
+def merge_attributes(openfisca_element, ipp_element):
+    for key, value in ipp_element.attrib.iteritems():
+        if key in openfisca_element.attrib:
+            if openfisca_element.attrib[key] != ipp_element.attrib[key]:
```

Comment:
Est ce que ce ne serait pas intéressant de décrire la situation où on ne remplace pas:
` (key in openfisca_element.attrib) and (openfisca_element.attrib[key] == ipp_element.attrib[key])`
et de dire qu'on remplace pour toutes les autres ?

Deuxième question : Pourquoi itérer sur key/value, si on n'utilise pas value ?